api/communication-api/src/main/java/com/cognifide/aet/communication/api/metadata/Statistics.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Automated Exploratory Tests
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.aet.communication.api.metadata;
+
+import java.io.Serializable;
+
+public class Statistics implements Serializable {
```

Comment:
Is there anything else that we can measure inside Statistics class in the future?
I'm asking because it looks like we have here the whole class to wrap single field with no additional reasons.